code/datums/spawners_menu.dm

```diff
@@ -55,17 +57,24 @@
 	if(!group_name || !(group_name in GLOB.mob_spawners))
 		return
 	var/list/spawnerlist = GLOB.mob_spawners[group_name]
+	for(var/obj/effect/mob_spawn/current_spawner in spawnerlist)
```

Comment:
```suggestion
	for(var/obj/effect/mob_spawn/current_spawner as anything in spawnerlist)
```
Does this need a type check?

Reply:
I checked, and only /mob_spawners are added to GLOB.mob_spawners, but UI code supports showing non-/mob_spawners that could end up in GLOB.mob_spawners, so it is making me a little uneasy.

Reply:
How could they end up in the list? Is that for sure, or something that might happen?
We don't want to have mixed type lists whenever possible, so if it happens that needs to be fixed.

Reply:
It might happen, but I guess its something that we will have to deal with when it actually happens. Commited.